CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## Unreleased
+### Fixed
+* Fixed an issue with assets saving Contentful::Management::File objects instead of it's JSON representation when using `--mark-processed`.
```

Comment:
"Fixed an issue in which assets savE Contentful::Management::File objects instead of as JSON when using `--mark-processed`."